src/commands/deploy.ts

```diff
@@ -419,18 +449,23 @@ To file a ticket, please head to: https://console.liara.ir/tickets`)
             if (release.state === 'FAILED') {
               this.spinner.fail();
               if(release.failReason) {
+                ipcSendLog({log: this.parseFailReason(release.failReason), state: 'build', status: 'error'})
                 return reject(new DeployException(this.parseFailReason(release.failReason)))
               }
+              ipcSendLog({log: 'Release failed.', state: 'build', status: 'error'})
               return reject(new Error('Release failed.'))
             }
 
             if (release.state === 'DEPLOYING' && !isDeploying) {
               isDeploying = true
+              ipcSendLog({log: 'Image pushed.', state: 'build', status: 'pending'})
               this.spinner.succeed('Image pushed.')
+              ipcSendLog({log: 'Creating a new release...', state: 'build', status: 'pending'})
```

Comment:
I guess we need to remove this line.

Reply:
Literally, I don't know why  
we are supposed to show users' new releases, aren't we??

Reply:
We're printing `Creating a new release...` log in 2 different lines. You can `ctrl + f` it.

Reply:
you were right... they were in differentness stage